env.sh

```diff
@@ -0,0 +1,40 @@
+# Script to populate the environment variables for CDK deployment/pgstac database ingestion
+
+# Usage: source populate_env.sh <env>
+# Valid environments: dev, stage (for now)
+
+devPGSecret=delta-backend-devv2/pgstac/c0b42a5a
+stagePGSecret=delta-backend-stagingv2/pgstac/5d4eb447
```

Comment:
🤔 I kind of think these identifiers should be left out of the public repository, especially since they could change, perhaps same with the identifiers below. Curious what @anayeaye @tracetechnical @alukach  think about this

Reply:
Absolutely not. Sensitive information like this should be fed in.
As it stands, this is handing out information to enable an attack.

Reply:
I don't personally think this is much of an issue.  This is the identifier for a secret, not the secret itself.  Does the identifier of sensitive information qualify as being _sensitive_?  At the end of the day, we will need to refer to the secrets in some way.  If we do think it should be changed, please provide a suggestion as to what would be a better solution.

Reply:
i've removed the file and added a sample file with placeholders in https://github.com/NASA-IMPACT/cloud-optimized-data-pipelines/pull/138/commits/d148772831fcfb59ecfbc300d1aa79ccad2dab34 and https://github.com/NASA-IMPACT/cloud-optimized-data-pipelines/pull/138/commits/43858cc7b32e532c6dfc30c45f0eff5a5da81c6c

Reply:
yeah, need to have them removed from the repo.